### PR TITLE
Alexp mcafee ESM Integration addition

### PR DESCRIPTION
**Pull request by apekarovsky**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added 3 commands to get/add data to/remove data from McAfee SIEM Watchlists.
Important functionality asked by customers/partners. It's similar to adding IP's/Hashes to 
## Screenshots

Screen Shot 2020-07-15 at 20 38 58

## Minimum version of Demisto
- [ X] 4.5.0
- [ X] 5.0.0
- [ X] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [X ] Tests - successfully completed the demisto-sdk lint tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

File: Packs/McAfee_ESM-v10/Integrations/McAfee_ESM-v10/McAfee_ESM-v10.py [hunk 1]
```diff
@@ -1,8 +1,7 @@
-import demistomock as demisto
+""" IMPORTS """
 from CommonServerPython import *
 from CommonServerUserPython import *
-
-""" IMPORTS """
+from demistomock import *
```
Comment (dantavori) [inline Packs/McAfee_ESM-v10/Integrations/McAfee_ESM-v10/McAfee_ESM-v10.py:4]: let's revert this change as we talked

File: Packs/McAfee_ESM-v10/Integrations/McAfee_ESM-v10/McAfee_ESM-v10.yml [hunk 2]
```diff
@@ -720,6 +720,49 @@ script:
...
+        description: The type of the watchlist
+        type: string
+      - contextPath: EsmWatchlists.ValuesNumber
+        description: The number of values in the watchlist
+        type: number
+      - contextPath: EsmWatchlists.IsDynamic
+        description: Is the watchlist dynamic
+        type: boolean
+      - contextPath: EsmWatchlists.IsActive
+        description: Is the watchlist active
```
Comment (dantavori) [inline Packs/McAfee_ESM-v10/Integrations/McAfee_ESM-v10/McAfee_ESM-v10.yml:739]:
```suggestion
      - contextPath: EsmWatchlist.ID
        description: The Id of the watchlist
        type: string
      - contextPath: EsmWatchlist.Type
        description: The type of the watchlist
        type: string
      - contextPath: EsmWatchlist.ValuesNumber
        description: The number of values in the watchlist
        type: number
      - contextPath: EsmWatchlist.IsDynamic
        description: Is the watchlist dynamic
        type: boolean
      - contextPath: EsmWatchlist.IsActive
        description: Is the watchlist active
```

File: Packs/McAfee_ESM-v10/Integrations/McAfee_ESM-v10/McAfee_ESM-v10.py [hunk 3]
```diff
@@ -848,6 +875,34 @@ def users_to_entry(users):
...
+        fixed_watchlists.append({
+            'ID': watchlist['id'],
+            'Name': watchlist['name'],
+            'Type': watchlist['type']['name'],
+            'ValuesNumber': watchlist['valueCount'],
+            'IsDynamic': watchlist['dynamic'],
+            'IsActive': watchlist['active'],
+        })
+
+    context = {'EsmWatchlists(val.ID && val.ID == obj.ID)': fixed_watchlists}
```
Comment (dantavori) [inline Packs/McAfee_ESM-v10/Integrations/McAfee_ESM-v10/McAfee_ESM-v10.py:895]:
```suggestion
    context = {'EsmWatchlist(val.ID && val.ID == obj.ID)': fixed_watchlists}
```

File: Packs/McAfee_ESM-v10/Integrations/McAfee_ESM-v10/McAfee_ESM-v10.yml [hunk 4]
```diff
@@ -720,6 +720,49 @@ script:
...
+    description: Gets the names of the watchlists
+  - name: esm-add-watchlist-values
+    arguments:
+      - name: id
+        required: true
+        description: ID of the watchlist we want to add values to. ID can be retrieved
+          with the "esm-get-watchlists" command.
+      - name: values
+        required: true
+        description: 'Coma separated values of the data we want to add to the watchlist.
```
Comment (dantavori) [inline Packs/McAfee_ESM-v10/Integrations/McAfee_ESM-v10/McAfee_ESM-v10.yml:750]:
```suggestion
        description: 'A comma-separated values of the data to be added to the watchlist.
```

File: Packs/McAfee_ESM-v10/Integrations/McAfee_ESM-v10/McAfee_ESM-v10.yml [hunk 5]
```diff
@@ -720,6 +720,49 @@ script:
...
+    description: 'Add values to an existing watchlist. '
+  - name: esm-remove-watchlist-values
+    arguments:
+      - name: id
+        required: true
+        description: ID of the watchlist we want to remove values from. ID can be retrieved
+          with the "esm-get-watchlists" command.
+      - name: values
+        required: true
+        description: 'Coma separated values of the data we want to remove from the watchlist.
```
Comment (dantavori) [inline Packs/McAfee_ESM-v10/Integrations/McAfee_ESM-v10/McAfee_ESM-v10.yml:762]:
```suggestion
        description: 'A comma-separated values of the data to be removed from the watchlist.
```

**Comment by anara123** [comment]
@dantavori what is the status here?

**Comment by apekarovsky** [comment]
> @dantavori what is the status here?

@anara123 It's on my side. I need to complete the test playbook and the documentation. Actually close to complete. Should be ready tomorrow.

**Comment by apekarovsky** [comment]
@dantavori done, let's do the demo.

File: Packs/McAfee_ESM-v10/TestPlaybooks/playbook-McAfeeESMTest.yml [hunk 6]
```diff
@@ -1,14 +1,15 @@
 id: McAfeeESMTest
-version: -1
+version: 3
```
Comment (dantavori) [inline Packs/McAfee_ESM-v10/TestPlaybooks/playbook-McAfeeESMTest.yml:2]:
```suggestion
version: -1
```

File: Packs/McAfee_ESM-v10/Integrations/McAfee_ESM-v10/README.md [hunk 7]
```diff
@@ -3253,4 +3256,275 @@
...
+<tr>
+<td style="width: 187px;">id</td>
+<td style="width: 452px;">
+ID of the watchlist we want to add values to. ID can be retrieved with the "esm-get-watchlists" command.</td>
+<td style="width: 101px;">Required</td>
+</tr>
+<tr>
+<td style="width: 187px;">values</td>
+<td style="width: 452px;">
+Coma separated values of the data we want to add to the watchlist. Example: "1.1.1.1,2.2.2.2". Please add the correct type of values to correct watchlist type.</td>
```
Comment (dantavori) [inline Packs/McAfee_ESM-v10/Integrations/McAfee_ESM-v10/README.md:3442]: Please make sure the changes were made in the descriptions are applied in the readme too
